### ISSUE TITLE
Constructor `New` should be `new`

### ISSUE THREAD

**Issue by hoodmane** [pyodide/webtypy#9]
I get the following mypy error:
```py
error: "type[AbortController]" has no attribute "new"  [attr-defined]
            self.abortcontroller = AbortController.new()
```
In webtypy's bindings I see:

```py
class AbortController:
    def New(self) -> AbortController: ...
    signal: AbortSignal
    def abort(self, reason: Any | None = None): ...
```
Here `New` should be `new`.

**Comment by hoodmane** [comment]
Actually it seems that there is a second problem which is that `new` needs to be a `@classmethod`

**Comment by simonegiacomelli** [comment]
I just published version 0.1.7 on pypi. 
Let me know if something is amiss

**Comment by hoodmane** [comment]
Thanks for the quick fix @simonegiacomelli! I should switch Pyodide over to using webtypy, it will turn up a bunch more problems. I will open more issues with some things.
